Organisation en dossiers

MaximePlancke opened the pull request.

MaximePlancke committed:
Organisation en dossiers

ArnaudSpanneut commented:
@MaximePlancke Pour les noms de dossier, favorise les minuscules

ArnaudSpanneut commented on add_friends.php:
```diff
@@ -62,18 +62,18 @@
 	<!-- On ouvre la fenêtre à la largeur de l'écran -->
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<link rel="stylesheet" type="text/css" href="style.css">
+	<link rel="stylesheet" type="text/css" href="Ressources/style.css">
```
@MaximePlancke Du coup minuscule dans tes liens aussi.

ArnaudSpanneut commented on connexion/connexion.php:
```diff
@@ -20,7 +20,7 @@
 	    $_SESSION['id'] = $user['id'];
 	    $_SESSION['pseudo'] = $pseudo;
-		header('Location:index.php');
+		header('Location:../index.php');
```
@MaximePlancke Change TOUT tes liens (css, include, require ect...) en lien absolu => http://goo.gl/XcD7M

MaximePlancke committed:
Premier changement chemin absolu

encore un problème de chemin pour chemin absolu pour les menus et la
bdd à voir demain

MaximePlancke committed:
Modification2 chemin absolu

MaximePlancke committed:
Modification3 chemin absolu

MaximePlancke committed:
Modif4 chemin absolu

MaximePlancke committed:
Modif5 chemin absolu

ArnaudSpanneut commented:
@MaximePlancke Garde une homogénéité dans tes nom de fichiers ex: add_friends / envoi_obj (à renommer en add_objective du coup)

MaximePlancke committed:
Modification des noms envoi_obj et new_obj pour une meilleure homogénéité

MaximePlancke committed:
Update add_friends.php

MaximePlancke committed:
Merge branch 'master' of https://github.com/MaximePlancke/Objectifs

Conflicts:
	user/add_friends.php

MaximePlancke committed:
Merge branch 'opti/organisation' of https://github.com/MaximePlancke/Objectifs into opti/organisation

Conflicts:
	user/add_friends.php

MaximePlancke committed:
Ajout de add_friends sur opti/orga

ArnaudSpanneut commented:
@MaximePlancke pour la requête amis, c'est pas la bonne manière, tu dois limiter un max le nombre de requêtes!
Tu dois enregistrer une fois et supprimer une fois. Et c'est dans la requête SELECT que tu dois tester dans les 2 sens

ArnaudSpanneut commented on user/add_friends.php:
```diff
@@ -0,0 +1,98 @@
+	$request->execute(array(
+		'id_friends_1' => $id_friends_2,
+	    'id_friends_2' => $id_friends_1));
+	$request->closeCursor();
```
@MaximePlancke Attention à l'indentation

MaximePlancke committed:
Optimisation de Add_friends, delete account et menu_left (Requêtes SQL)

MaximePlancke committed:
Ajout de la fonction "supprimer objectifs" avec vérification de la session pour ne pas pouvoir supprimer les objectifs des autres

ArnaudSpanneut commented on user/add_friends.php:
```diff
@@ -0,0 +1,86 @@
+				<h4><?php echo $datas['pseudo'];?>
+					<a href="/user/add_friends.php?id_friends_1=<?php echo $_SESSION['id']; ?>&amp;id_friends_2=<?php echo $datas['id']; ?>&amp;fonction=add">  Ajouter</a>
+					<a href="/user/add_friends.php?id_friends_1=<?php echo $_SESSION['id']; ?>&amp;id_friends_2=<?php echo $datas['id']; ?>&amp;fonction=delete">  Supprimer</a>
+				</h4>
```
@MaximePlancke Inclus ça dans ton < li > et le h4 est pas nécessaire. de plus il faut que tu fasses avec des < form > et à la place de mettre des liens, mettre des < submit > pour faire du POST

ArnaudSpanneut commented on user/add_friends.php:
```diff
@@ -0,0 +1,86 @@
+	</section>
+	<?php include($_SERVER['DOCUMENT_ROOT']."/menus/footer.php"); ?>
+	</div>
+	<!-- Intégration de la libraire de composants du Bootstrap -->
```
@MaximePlancke Commentaires en anglais et en php

ArnaudSpanneut commented on user/add_friends.php:
```diff
@@ -0,0 +1,86 @@
+	<?php include($_SERVER['DOCUMENT_ROOT']."/menus/menu_top.php"); ?>
+	<section id="main_page" class="row-fluid">
+	<?php include($_SERVER['DOCUMENT_ROOT']."/menus/menu_left.php"); ?>
+	<!-- On recupère la requete dans menu_left pour afficher la liste des utilisateurs.  -->
```
@MaximePlancke Commentaires en anglais et en php

MaximePlancke committed:
Changement commentaire php / un requete pour friends / ajout Cascade et foreign key dans la BBD / méthode POST dans add_friends / changement des messages

MaximePlancke committed:
Dernier commentaire en francais traduit en anglais / ajout de add_objective dans new_objective et verification des variables

MaximePlancke committed:
petite modif sur l'include du footer

MaximePlancke committed:
Revert "petite modif sur l'include du footer"
